sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeysTestBase.cs

```diff
@@ -192,6 +196,34 @@ protected void AssertKeyPropertiesEqual(KeyProperties exp, KeyProperties act)
             AssertAreEqual(exp.Tags, act.Tags);
         }
 
+        protected static void AssertAreEqual(byte[] exp, byte[] act)
+        {
+            static byte[] TrimStart(byte[] buf)
+            {
+                int start = 0;
+                for (; start < buf.Length && buf[start] == 0; start++)
+                    ;
+
+                if (start != 0)
+                {
+                    return buf.AsSpan().Slice(start, buf.Length - start).ToArray();
+                }
+
+                return buf;
+            }
+
+            if (exp is null && act is null)
+                return;
+
+            if (exp?.Length != act?.Length)
+            {
+                exp = TrimStart(exp);
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.memoryextensions.trim?view=net-5.0#System_MemoryExtensions_Trim__1_System_ReadOnlySpan___0____0_ ?

Reply:
Not in net461. Actually, in this case, wasn't added till netcoreapp3.0. There were other options, but all started no later than net472.